apps/prairielearn/src/pages/instructorAssessmentQuestions/instructorAssessmentQuestions.html.ts

```diff
@@ -0,0 +1,399 @@
+                    <strong>All ungraded attempts will be lost.</strong>
+                  </p>
+                  <p>Students will receive a new variant the next time they view this question.</p>
+                </div>
+                <div class="modal-footer">
+                  <form name="reset-question-variants-form" method="POST">
+                    <input type="hidden" name="__action" value="reset_question_variants" />
+                    <input type="hidden" name="__csrf_token" value="${resLocals.__csrf_token}" />
+                    <input
+                      type="hidden"
+                      name="unsafe_assessment_question_id"
+                      class="js-assessment-question-id"
+                      value=""
+                    />
+                    <button type="button" class="btn btn-secondary" data-dismiss="modal">
+                      Cancel
+                    </button>
+                    <button type="submit" class="btn btn-danger">Reset question variants</button>
+                  </form>
+                </div>
+              </div>
+            </div>
+          </div>
+
+          ${renderEjs(
+            __filename,
+            "<%- include('../partials/assessmentSyncErrorsAndWarnings'); %>",
+            resLocals,
+          )}
+
+          <div class="card mb-4">
+            <div class="card-header bg-primary text-white d-flex align-items-center">
+              ${resLocals.assessment_set.name} ${resLocals.assessment.number}: Questions
+            </div>
+            ${AssessmentQuestionsTable({
+              questions,
+              assessmentType: resLocals.assessment.type,
+              urlPrefix: resLocals.urlPrefix,
+              hasCourseInstancePermissionEdit:
+                resLocals.authz_data.has_course_instance_permission_edit,
+            })}
+          </div>
+        </main>
+      </body>
+    </html>
+  `.toString();
+}
+
+function AssessmentQuestionsTable({
+  questions,
+  urlPrefix,
+  assessmentType,
+  hasCourseInstancePermissionEdit,
+}: {
+  questions: AssessmentQuestionRow[];
+  assessmentType: string;
+  urlPrefix: string;
+  hasCourseInstancePermissionEdit: boolean;
+}) {
+  let nTableCols = 11;
+
+  // If at least one question has a nonzero unlock score, display the Advance Score column
+  const showAdvanceScorePercCol =
+    questions.filter((q) => {
+      return q.assessment_question_advance_score_perc !== 0;
+    }).length >= 1;
+  if (showAdvanceScorePercCol) nTableCols++;
+
+  function maxPoints({ max_auto_points, max_manual_points, points_list, init_points }) {
+    if (max_auto_points || !max_manual_points) {
+      if (assessmentType === 'Exam') {
+        return `${(points_list || [max_manual_points]).map((p) => p - max_manual_points)}`;
+      }
+      if (assessmentType === 'Homework') {
+        return `${init_points - max_manual_points}/${max_auto_points}`;
+      } else {
+        return html`&mdash;`;
+      }
+    }
```

Comment:
I think you converted this wrong. If you take a look at the original code, I believe the` &mdash` case should be what we render if `max_auto_points || !max_auto_points` is `false`.

Reply:
You are right. Updated to this to render `&mdash` if `max_auto_points || !max_auto_points` is false in [d833d29](https://github.com/PrairieLearn/PrairieLearn/pull/9680/commits/d833d29e6af8ffd2dbe97453d413dfd718f7c289).